pkg/utils/release/controller.go

```diff
@@ -44,6 +45,33 @@ func (s *SuiteController) CreateSnapshot(name string, namespace string, applicat
 	return snapshot, s.KubeRest().Create(context.TODO(), snapshot)
 }
 
+func (s *SuiteController) GetSnapshotInNamespace(namespace, componentName string) (*appstudioApi.Snapshot, error) {
```

Comment:
I think like `GetSnapshotByComponent` would be a better name for what this does, but just a thought

Reply:
agree, will change it